### DISCUSSION TITLE
Form Slider? Found in Github

### DISCUSSION THREAD

**Discussion by gaa23** [Budibase/budibase#5164] [General]
It looks like this is code for a slider in the form components: https://github.com/Budibase/bbui/blob/master/src/Form/Slider.svelte

Am I missing something? I can't see this inside the builder.

**Comment by aptkingston** [comment]
Hey @gaa23. The `bbui` repository is our library of form components that we use in the platform as a whole, but we haven't added the slider as a component to be used inside user apps. The code for components available inside apps is inside https://github.com/Budibase/budibase/tree/develop/packages/client/src/components/app. Feel free to open a feature request issue if you'd like this added as a component though!
